package.json

```diff
@@ -77,9 +77,6 @@
   "demo": "https://jsfiddle.net/allenluce/c49r5b4b/",
   "browsers": true,
   "badges": {
-    "list": [
-      "travisci",
-      "npmversion"
-    ]
+    "list": ["travisci", "npmversion"]
```

Comment:
Can you revert this formatting change?

Reply:
For some reason, it gave me an error when it was formatted like what was in red. I'll try and put it back and see if it has no errors.

Reply:
I put it back, and it comes up with this error:
Incorrect type. Expected "array".

Reply:
Can you paste the command you're running and the error output?